lib/tasks/cards.rake

```diff
@@ -197,6 +197,33 @@ namespace :cards do
         new_card.performs_trace = true
       end
 
+      if new_card.text && (new_card.text.include?("When you install #{new_card.title}") || new_card.text.include?('When you install this') || new_card.text.include?("#{new_card.title} is installed")) # rubocop:disable Layout/LineLength
+        new_card.install_effect = true
+      end
+
+      new_card.charge = true if new_card.text&.include?('charge')
+
+      new_card.has_gain_click = true if new_card.text&.include?('gain [click]')
+
+      new_card.mark = true if new_card.text&.include?('your mark')
+
+      if new_card.card_type_id == 'agenda' && new_card.text && (new_card.text.include?('when you score') || new_card.text.include?('When you score') || new_card.text.include?('score this agenda') || new_card.text.include?('scored or stolen')) # rubocop:disable Layout/LineLength
```

Comment:
up to you if you want to do this as a follow-up, but there are non-agendas also have steal and/or score effects.